core/src/main/java/com/exadel/etoolbox/linkinspector/core/services/data/impl/DataFeedServiceImpl.java

```diff
@@ -198,7 +198,7 @@ public void deleteDataFeed() throws DataFeedException {
             gridResourcesCache.clearCache();
             serviceResourceResolver.commit();
         } catch (PersistenceException e) {
-            LOG.error("Failed to delete data feed", e);
+            LOG.warn("Failed to delete data feed", e);
```

Comment:
Same here. We are not going to just suffocate all the errors in general) Every Java project has some exceptions to log as errors

Reply:
Please reduce the "not-an-error" logging practices to the only error messages that may come in masses, like, in hundreds. These are messages generated by link resolvers. Not by the rest of the code. On the contrary, if the rest of the code (like, "usual" services or sling models, etc.) fail, we need to know this, because it may tell about the malfunction of the solution

Reply:
done